### ISSUE TITLE
console.log didn't work properly in TM Beta

### ISSUE THREAD

**Issue by langren1353** [Tampermonkey/tampermonkey#1387]
(Please fill out the issue template with your details)

## Expected Behavior
console.log works find, log the message at console

## Actual Behavior
no log exist. Even run log with GM_log, there is no log

## Specifications

- Chromium: 86.0.4240.198
- TM: 4.14.6149 Beta is Wrong
- TM: 4.13 Release is OK
- OS: Win

## Script

(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name         Log Test
// @namespace    Looog
// @version      0.1
// @description  Log Test...
// @author       ME
// @include      *
// @icon         https://www.google.com/s2/favicons?sz=64&domain=github.com
// @grant        GM_log
// ==/UserScript==

debugger
console.log('GM_log');
GM_log("this is a log, but I can't find it");
debugger
```

**Comment by 7nik** [comment]
Works for me, TM 4.14.6149, Chromium 96.0.4664.93.
Does the script even run for you? Or maybe something blocks logging?

**Comment by langren1353** [comment]
I tried two browser with Chromium 86.0.4240.198 it failed
Then I tried Edge 96.0.1054.53 it works.
-------------------------------------------------
Then I tried with Chrome:
86.0.4240.198: no log
90.0.4430.212: no log
96.0.4664.110: log exist
.........it is so confusing

**Comment by derjanb** [comment]
@langren1353  I can reproduce this with Chromium 75.0.3768.0  🤔

@7nik Thanks for your help all along.

**Comment by langren1353** [comment]
> @langren1353 I can reproduce this with Chromium 75.0.3768.0 🤔
> 
> @7nik Thanks for your help all along.

I tried with Chrome 75.0.3770.142, it still has no log.
Chrome Version List：[https://423down.lanzouo.com/b112274](https://423down.lanzouo.com/b112274) ,download it and unzip with 7zip or something, don't click it just unzip it.
And will you fix this on Beta version, not on Release version?

**Comment by derjanb** [comment]
Should be fixed at 4.14.6150 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_4_14_6150.crx))

**Comment by langren1353** [comment]
> Should be fixed at 4.14.6150 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_4_14_6150.crx))

It works very well. Thank you for your constant dedication on TamperMonkey support. Thanks again
